src/pytrajplot/plotting/analyse_trajectories.py

```diff
@@ -82,14 +82,17 @@ def _get_traj_dict(data, number_of_trajectories=None, traj_length=None) -> dict:
 
 def _check_dateline_crossing(
     lon: pd.Series,
+    number_of_times: int,
```

Comment:
Consider renaming this variable. Maybe `time_steps`?
`_check_dateline_crossing(lon, number_of_times=5)` sounds like the dateline crossing is checked 5 times.

Reply:
Sounds good 👍